Offline calibration refactor docs update (#238)
pupil-labs/pupil-docs: user-docs/pupil-player.md

Alternatively, one can run the pupil detection post-hoc.

#### Offline Pupil Detector
The `Offline Pupil Detector` plugin can be used with any dataset where eye videos were recorded. 
The plugin tries to load the eye videos, and runs the pupil detection algorithm in separate processes.
This plugin is especially relevant for recordings made with Pupil Mobile, because Pupil Mobile does not perform any pupil detection or gaze estimation on the Android device.
This plugin is available starting with Pupil Player `v0.9.13`.

The `Detection Method` selector sets the detection algorithm to either `2d` or `3d` detection (see [the section on Pupil Detection](#pupil-detection) for details).
The `Redetect` button restarts the detection procedure.
You can use the `Offline Pupil Detector` plugin to debug, improve, and gain insight into the pupil detection process.

### Gaze Data And Post-hoc Calibration
By default, Player starts with the `Gaze From Recording` plugin that tries to load gaze positions that were detected and stored during a Pupil Capture recording.

> Alternatively, one can run the calibration process post-hoc.

Reviewer: ```suggestion
Alternatively, one can run the gaze mapping process post-hoc.
```

#### Offline Calibration
The `Offline Calibration` plugin enables you to calibrate, map, and validate gaze post-hoc and is available starting with Pupil Player `v0.9.13`.
It can be used on any Pupil dataset.

The workflow is separated into three steps, each with its own submenu: Reference Locations, Calibrations, and Gaze Mappers.

> Reference locations are points within the recording that are known to have been fixated by the subject.

Reviewer 2: Reference locations are points within the world 
```suggestion
Reference locations are points within the recorded world video that are known to have been fixated on by the participant/subject.
```

They can either be automatically detected or manually annotated:

1. `Detect Circle Markers in Recording`: This button starts the automatic detection of [circular calibration markers](#calibration-methods) within the world video. The progress is visualized in the plugin's timeline.
2. `Manual Edit Mode`: When this option is enabled, you can add new locations as well as correct and delete existing ones. There can only be one location per world frame.

As in Capture, one can have more than one calibration per recording.

> A calibration on its own does not result in gaze data, yet.

Reviewer 2: ```suggestion
A calibration on its own does not result in gaze data.
```

> It rather contains the required parameters to map pupil to gaze data.

Reviewer 2: ```suggestion
Rather, it contains the required parameters to map pupil to gaze data.
```

Each has the following properties:

- `Name`: Used to correctly select a calibration for each gaze mapper (see below).

> - `Mapping Method`: `2d` uses polynomial regression, or `3d` uses bundle adjustment calibration

Reviewer 2: ```suggestion
- `Mapping Method`: `2d` uses polynomial regression, or `3d` uses bundle adjustment calibration.
```
